charts/synthetics-private-location/README.md

```diff
@@ -1,6 +1,6 @@
 # Datadog Synthetics Private Location
 
-![Version: 0.13.3](https://img.shields.io/badge/Version-0.13.3-informational?style=flat-square) ![AppVersion: 1.22.0](https://img.shields.io/badge/AppVersion-1.22.0-informational?style=flat-square)
+![Version: 0.13.4](https://img.shields.io/badge/Version-0.13.3-informational?style=flat-square) ![AppVersion: 1.23.0](https://img.shields.io/badge/AppVersion-1.23.0-informational?style=flat-square)
```

Comment:
The badge link still points to `0.13.3`. Running `.github/helm-docs.sh` should fix this.